Index: debian-3.55/debian.cabal
===================================================================
--- debian-3.55.orig/debian.cabal	2011-06-03 21:02:01.000000000 -0300
+++ debian-3.55/debian.cabal	2011-06-03 21:03:43.000000000 -0300
@@ -22,7 +22,7 @@
 Library
  Build-Depends:  base >= 4 && < 5, bytestring, containers, directory,
                  filepath, mtl, network, old-locale, parsec >= 2 && <4, pretty, process,
-                 regex-tdfa, regex-compat, time, unix, bzlib, HaXml == 1.20.*, Unixutils >= 1.32, zlib, HUnit,
+                 regex-tdfa, regex-compat, time, unix, bzlib, HaXml == 1.20.*, Unixutils >= 1.32, zlib,
                  utf8-string
  ghc-options: -W -Wall
  if flag(cabal19)
@@ -65,11 +65,7 @@
         Distribution.Package.Debian.Bundled,
         Distribution.Package.Debian.Setup,
         Distribution.Package.Debian.Main,
-        Distribution.Package.Debian,
-        Test.Changes,
-        Test.Dependencies,
-        Test.SourcesList,
-        Test.Versions
+        Distribution.Package.Debian
 
 Executable fakechanges
  Main-is: utils/FakeChanges.hs
